src/styles/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-color);
+}
+
+.card__button-buy {
+  display: flex;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+
+  align-items: center;
+  color: var(--button-additional-color);
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  border: 1px solid transparent;
+  background-color: var(--button-color);
+  border-radius: 5px;
+}
+
+.card__button-buy:hover {
+  background-color: var(--button-additional-color);
+  color: var(--button-color);
+  border-color: var(--button-color);
+  transition: 0.3s;
```

Comment:
move it to the parent selector then it will work not only on hover

Reply:
Ok. But in what situations do you also need transition except hover?

Reply:
mouse over and mouse out

Reply:
You can test it just moving `transition` from `.btn: hover` to `.btn`

Reply:
Ok. I made this correction, thx. I dont know JS so deeply so far.